scripts/zika_utils.py

```diff
@@ -72,20 +72,51 @@ def __init__(self, log, log_filename, lims, currentStep):
         sys.stderr.write("ERROR: Check log for more info.")
         sys.exit(2)
 
-def fetch_sample_data(currentStep, to_fetch):
+
+def fetch_output_udfs(currentStep):
```

Comment:
`fetch_output_udfs` and `assert_udfs` are not in use any more. Considering deleting them?

Reply:
[c415a87](https://github.com/SciLifeLab/scilifelab_epps/pull/286/commits/c415a87cb7ded6822d6251780e1a911d4b670714)